packages/client/subscriptions/NotificationSubscription.ts

```diff
@@ -257,6 +257,8 @@ const NotificationSubscription = (
           break
         case 'UpdateFeatureFlagPayload':
           break
+        case 'AuthTokenPayload':
```

Comment:
This fixes `NotificationSubscription case fail AuthTokenPayload` error showing in the logs. @mattkrick is there anything specific app should do here? just want to make sure it's not a bug and I'm not silencing it

Reply:
nope, this is just a reeeeeally old pattern that we haven't replaced yet. all the other subscriptions use a lookup table.